web/index.php

```diff
@@ -2,14 +2,13 @@
 	$uri=trim($_SERVER['REQUEST_URI'],'/');
 	$uri=urldecode($uri);
 
-	if (!$uri)
-	{
+	if (!$uri) {
 	  echo file_get_contents(__DIR__.'/homepage.html');
 	  exit;
 	}
-	 
+
+	echo file_get_contents(__DIR__.'/interstitial.html');
```

Comment:
im confused about this. it looks like you're opening and closing the `<html>` tag here, and then adding more html below (the `echo <meta...` line). does that work? doesnt seem like valid html.

Reply:
I actually have no idea. It seemed like this would work but my understanding of PHP is pretty novice.

Reply:
@lyoshenka we discussed above